<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnsembleAnalyzer</class>
 <widget class="QMainWindow" name="EnsembleAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ensemble Analyzer</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="SearchTermsLayout"/>
    </item>
    <item row="0" column="2" rowspan="11">
     <widget class="QTableWidget" name="Table"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="AddSearchTermButton">
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Add another search term</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="QueryButton">
      <property name="maximumSize">
       <size>
        <width>132</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Perform query</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="ChangeInputButton">
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Change input directory</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="InputButtonLabel">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="ChangeScriptButton">
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Change script directory</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="ScriptButtonLabel">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="ChangeOutputButton">
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Change output directory</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="OutputButtonLabel">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="ParametersLabel">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Input script parameters</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="ScriptLabel">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Select script</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="ScriptParameters">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="ScriptSelector">
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>26</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="RunScriptButton">
      <property name="maximumSize">
       <size>
        <width>106</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Run script</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
